spec/features/create_photos_spec.rb

```diff
@@ -0,0 +1,8 @@
+feature 'Photo' do
+  it 'can create a photo' do
+    visit 'photos/new'
+    attach_file('photo_image', Rails.root + 'spec/files/test.png', visible: false)
+    click_button 'Upload'
+    expect(Photo.count).to eq 1
+  end
```

Comment:
check the bad path as well - what if a user tries to upload something that is not a photo? Are there any edge cases you can think of?